gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -27,5 +32,78 @@
 
   public static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
 
-  // TODO(ajaykannan): Fix me! Add in missing methods.
+  /**
+   * Create a new project.
+   *
+   * @return ProjectInfo object representing the new project's metadata.  The returned object will
+   * include additional read-only information, namely project number, lifecycle state, and creation
+   * time.
+   * @throws ResourceManagerException upon failure
+   */
+  ProjectInfo create(ProjectInfo project);
+
+  /**
+   * Delete the requested project.
```

Comment:
Should we say here that this only requests deletion? And maybe that the deletion request can be undone with `#undelete()`?